modules/ocf_irc/files/anope.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=A set of IRC Services
+After=network-online.target inspircd.service
+
+[Service]
+User=irc
+Group=irc
+Restart=always
+# Run ExecStartPre with root permissions to be able to create pid directory
+PermissionsStartOnly=true
+ExecStartPre=/bin/mkdir -p /var/run/anope/
+ExecStartPre=/bin/chown -R irc:irc /var/run/anope/
```

Comment:
If the original script does that then it seems more reasonable to copy, I'll do that.